### PR TITLE
Update ALL SCRIPTS.vbs

### PR DESCRIPTION
**Pull request by KallistaImdieke**
added button for new mod client contact note.

### CODE REVIEW

File: ALL SCRIPTS.vbs [hunk 1]
```diff
@@ -273,6 +273,15 @@ cs_scripts_array(script_num).agencies_that_use		= "BELTRAMI, HENNEPIN, MILLE LAC
 script_num = script_num + 1
 ReDim Preserve cs_scripts_array(script_num)
 SET cs_scripts_array(script_num) = NEW cs_script
+cs_scripts_array(script_num).script_name		= "MOD CAAD NOTE: CONTACT CHECKLIST"
+cs_scripts_array(script_num).script_type		= "NOTES"
+cs_scripts_array(script_num).file_name			= "NOTES - MOD CAAD NOTE: CONTACT CHECKLIST.vbs"
+cs_scripts_array(script_num).description		= "NEW 06/2016!! Creates CAAD note for recording contact with Client regarding possible Mod."
+cs_scripts_array(script_num).agencies_that_use		= "BELTRAMI, HENNEPIN, MILLE LACS, OLMSTED, RAMSEY, RENVILLE, SCOTT, STEARNS, WASHINGTON, WRIGHT"
```
Comment (RobertFewins-Kalb) [inline ALL SCRIPTS.vbs]: You left Anoka and Dakota off the list.

**Comment by RobertFewins-Kalb** [comment]
@KallistaImdieke you still need to pull in the script. If I accept this pull request, there will be a button to no where.

**Comment by RobertFewins-Kalb** [comment]
@KallistaImdieke, reminder, we still need a script file for this pull request.

**Comment by KallistaImdieke** [comment]
I am trying to find help on how to do this, I will get it done for you thanks!

**Comment by KallistaImdieke** [comment]
added the code. Hopefully this will resolve this. Thanks!

**Comment by RobertFewins-Kalb** [comment]
Looks great!! Nicely done!

**Pull request merged**